### ISSUE TITLE
CheckBox: Fluent theme

### ISSUE THREAD

**Issue by Vitalius1** [OfficeDev/office-ui-fabric-react#6706]

**Comment by Vitalius1** [comment]
Updated with #6650

**Comment by mikewheaton** [comment]
Assigning to @betrue-final-final for design review.

**Comment by betrue-final-final** [comment]
Hey @Vitalius1,
Looks like there are issues in checkboxes:

1. hover border color still uses #212121, should be gray160
2. checked uses an odd hex value. Not sure if this is an issue with themePrimary, but it reads as #0378D4, and should be #0078D4

**Comment by betrue-final-final** [comment]
Hey this looks great without the bug! Just one thing and we're good!
Disabled text should be gray90. This is a global change we made so themes would work properly.

**Comment by Vitalius1** [comment]
@betrue-final-final wait... does this apply to all disabled components? The toolkit still uses gray60 from what I've seen. Let's talk offline to make sure we have it right for all other components too.

**Comment by mikewheaton** [comment]
And if it is a global change, let's try to apply it via an update to the theme's `semanticColors` object instead of overriding it separately in each place.

**Comment by Vitalius1** [comment]
@mikewheaton 
I don't think the issue was in the theme's semanticColors... It's just I was blindly following the toolkit when it needed to be updated 😄 by Ben. Now I have the correct colors.
@betrue-final-final 
Should be fixed after #6739 gets a merge into master. Let me know if I still didn't get it right 😊

**Comment by betrue-final-final** [comment]
Looks good.

**Comment by marygans** [comment]
Need to update indeterminant state as stated here in this [open issue](https://github.com/OfficeDev/office-ui-fabric-react/issues/5390)

**Comment by maxwellred** [comment]
Closing this as I added indeterminate state in a recent PR #9815.
